packages/lesswrong/components/users/UsersEditForm.jsx

```diff
@@ -45,7 +45,7 @@ const UsersEditForm = (props) => {
     <div className="page users-edit-form">
       <Typography variant="display2" className={classes.header}><FormattedMessage id="users.edit_account"/></Typography>
       <Button color="secondary" variant="outlined" className={classes.resetButton } onClick={() => Accounts.forgotPassword({ email: props.currentUser.email },
-          (error) => props.flash(error ? error.reason : "Sent password reset email to " + props.currentUser.email))
+          (error) => props.flash({ messageString: error ? error.reason : "Sent password reset email to " + props.currentUser.email }))
```

Comment:
does this require a type? (the other updates included "type: success", and this looks like it's supposed to have type: error", unless that comes automatically?)

Reply:
The type is passed around in such a way that we could vary the styles of the message based on it, but we don't. I think some other usages (not changed) are also missing `type`. Omitting it definitely doesn't break the message display (I tested this message in particular).